src/commands/upload_file.ts

```diff
@@ -121,6 +123,34 @@ function getSingleFile(parameters: ParametersHelper, parentFolderId: FolderID):
 	return [singleParameter];
 }
 
+async function getRemoteFile(
+	parameters: ParametersHelper,
+	parentFolderId: FolderID
+): Promise<UploadPathParameter[] | undefined> {
+	const remoteFilePath = parameters.getParameterValue(RemotePathParameter);
+
+	if (!remoteFilePath) {
+		return undefined;
+	}
+
+	const tempFolder = getTempFolder();
+
+	const localFilePath = await download(remoteFilePath, tempFolder, (downloadProgress: number) => {
+		console.error('Downloading file...', downloadProgress);
```

Comment:
can we check for `ARDRIVE_PROGRESS_LOG` env variable to display progress?
Also, it would be good to use "process.stderr.write(\`Downloading file... ${downloadProgress}\r\`)" so it will replace the current terminal line and basically just use one line.